package.json

```diff
@@ -49,10 +47,13 @@
     "node-slackr": "0.1.0",
     "octonode": "^0.7.4",
     "optimist": "0.2.0",
-    "rewire": "^2.3.4",
+    "passport": "^0.3.2",
```

Comment:
`passport` and `passport-google-auth` should also prolly be pinned instead of prefixed with ^.